pkg/load/load.go

```diff
@@ -232,3 +233,34 @@ func loadWorkflow(bytes []byte) (string, string, api.MultiStageTestConfiguration
 	}
 	return workflow.Workflow.As, workflow.Workflow.Documentation, workflow.Workflow.Steps, nil
 }
+
+func ClusterProfilesConfig(configPath string) (api.ClusterProfilesList, error) {
+	configContents, err := os.ReadFile(configPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read cluster profiles config: %w", err)
+	}
+
+	var profileOwnersList api.ClusterProfilesList
+	if err = yaml.Unmarshal(configContents, &profileOwnersList); err != nil {
+		return nil, fmt.Errorf("failed to unmarshall cluster profiles config: %w", err)
+	}
+
+	// The following code can be erased once profiles are completely moved
```

Comment:
This comment should be prefixed with `TODO`. We should also make sure we have a card somewhere to followup on that

Reply:
Good point. I have a card: https://issues.redhat.com/browse/DPTP-3642